plugins/chat/app/controllers/chat/chat_controller.rb

```diff
@@ -148,7 +148,7 @@ def create_message
         )
 
       Chat::Publisher.publish_user_tracking_state!(current_user, @chat_channel, message)
-      render json: success_json
+      render json: success_json.merge(message_id: message.id)
```

Comment:
I wonder we shouldn't just return a serialized message here, but we can defer on this

Reply:
Yeah I was surprised we only did OK but I agree we can defer